src/pages/on-chain-governance/[proposalId]/index.page.tsx

```diff
@@ -101,6 +114,7 @@ function ProposalDetail({
             "bg-primary-50 text-primary-500 rounded-sm text-sm font-medium py-2 px-18 self-auto md:self-start lg:self-end grow w-full md:w-[200px] lg:w-auto"
```

Comment:
think will need to check active and hover state for this

Reply:
added hover state, but focus state is the same as default